### ISSUE TITLE
Renewing address of remote actor after process restart

### ISSUE THREAD

**Issue by HoneyryderChuck** [celluloid/dcell#70]
I have the following use case:
1. Process1 creates cell1 accessible under the address tcp://127.0.0.1:8887
2. Process2 creates cell2 accessible under the address tcp://127.0.0.1:8888
3. cell2 sends message to cell1, cell1 responds successfully
4. Process1 goes down, cell1 not accessible anymore
5. cell2 sends message to cell1, infinite loop

Now, the infinite loop is already an issue, but now i have the following divergent cases (just for the sake of the example, assume step 5 didn't occur and process2 is still standing):

---

6.1.1 Process1 restarts, cell1 accessible under the address tcp://127.0.0.1:8887
6.1.2 cell2 sends message to cell1, cell1 responds successfully

---

6.2.1 Process1 restarts, cell1 accessible under the address tcp://127.0.0.1:8889
6.2.2 cell2 sends message to cell1, infinite loop

---

So, bottom line is, Process2 still had the address of cell1 as being under tcp://127.0.0.1:8887, even though process went down. After Process1 restarted, it renewed its address in the registration server (i'm using zookeeper, by the way), but this has not been propagated to the other processes. So, unless the address is exactly the same as before, communication between remote actors will cease, unless both processes are restarted. 

In a scenario with long running processes and first-open-port-in-range-selection policy, this is doomed to fail. My question to is: does DCell provide already some workaround for the issue, and if not, where should one start investigating a possible solution?
